projects/igniteui-angular/migrations/update-11_1_0/index.ts

```diff
@@ -4,12 +4,42 @@ import {
     Tree
 } from '@angular-devkit/schematics';
 import { UpdateChanges } from '../common/UpdateChanges';
+import {
+    findMatches,
+    replaceMatch
+} from '../common/tsUtils';
 
 const version = '11.1.0';
 
 export default (): Rule => (host: Tree, context: SchematicContext) => {
     context.logger.info(`Applying migration for Ignite UI for Angular to version ${version}`);
 
     const update = new UpdateChanges(__dirname, host, context);
+    const tsFiles = update.tsFiles;
+    const targetEnum = 'GridPagingMode';
+    const changes = [
+        { member: 'remote', replaceWith: 'Remote', definedIn: [targetEnum] },
+        { member: 'local', replaceWith: 'Local', definedIn: [targetEnum] }
+    ];
+    for (const entryPath of tsFiles) {
+        const ls = update.getDefaultLanguageService(entryPath);
+        let content = host.read(entryPath).toString();
+        const matches = [];
```

Comment:
This `matches` array seems kind of redundant to me now. Instead of pushing into the array we can simply loop over the indices returned by `findMatches` in the `for (const change of changes)` loop. And then just move all of the following logic in that loop as well. This way we won't have to loop over the same match positions twice.